app/documents/order_matrix.rb

```diff
@@ -51,12 +55,14 @@ def body
 
       # Make order_articles header
       header = [""]
+      
       for header_article in current_order_articles
         name = header_article.article.name.gsub(/[-\/]/, " ").gsub(".", ". ")
-        name = name.split.collect { |w| w.truncate(8) }.join(" ")
-        header << name.truncate(30)
+        name = name.split.collect { |w| w.truncate(20) }.join(" ")
+        header << name.truncate(35)+' - ('+header_article.article.unit+')'
```

Comment:
`header << "#{name.truncate(35)} - (#{header_article.article.unit})"` might be easier to read